src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -386,7 +387,7 @@ export const View = (props: Props) => {
               typeof props.totalNumberOfPerformedScans === "undefined" ||
               props.totalNumberOfPerformedScans <
                 CONST_ONEREP_MAX_SCANS_THRESHOLD ? (
-                <a
+                <Link
```

Comment:
@flozia This came out of a new linting error, presumably a bugfix in a parser or something. You added the initial `<a>` - do you remember if that was a conscious decision, or an oversight?